framework/caching/DbCache.php

```diff
@@ -256,7 +256,7 @@ protected function deleteValue($key)
      */
     public function gc($force = false)
     {
-        if ($force || mt_rand(0, 1000000) < $this->gcProbability) {
+        if ($force || random_int(0, 1000000) < $this->gcProbability) {
```

Comment:
I am unsure it make sense to change `gc`. It has nothing to do with security or cryptographics.
Replacing `mt_rand` with `random_int` will degrade performance without purpose here.